src/main/java/org/springframework/social/partnercenter/api/PartnerCenterResponse.java

```diff
@@ -58,12 +61,23 @@ public PartnerCenterResponse setLinks(ResourceLinks links) {
 	}
 
 	public String getContinuationToken() {
-		return continuationToken;
+		return extractContinuationToken().orElse(null);
```

Comment:
Does this work for the customer list? AFAIK the customer list response still has the old parameter.